server/bin/pbench-server-prep-shim-002.py

```diff
@@ -151,6 +171,14 @@ def process_tb(config, logger, receive_dir, qdir_md5, duplicates, errors):
             nquarantined += 1
             continue
 
+        try:
+            dataset.md5 = archive_md5_hex_value
```

Comment:
This should be an `if dataset:` ... oops.

It was awkward having to split this, but I needed to create the dataset before the first quarantine call in order to avoid having quarantine need to be able to create a dataset, and that's too early to know the md5.